conanfile.py

```diff
@@ -10,9 +10,15 @@ class ProtobufConan(ConanFile):
     version = "2.6.1"
     url = "https://github.com/memsharded/conan-protobuf.git"
     settings = "os", "compiler", "build_type", "arch"
-    exports = "CMakeLists.txt", "*.cmake", "extract_includes.bat.in"
+    exports = "CMakeLists.txt", "*.cmake", "extract_includes.bat.in", "change_dylib_names.sh"
     options = {"static": [True, False]}
     default_options = "static=False"
+    requires = "zlib/1.2.8@lasote/stable"
+    generators = "cmake"
+    license = "https://github.com/google/protobuf/blob/v2.6.1/LICENSE"
+
+    def config(self):
```

Comment:
Hi @kbinani 

I was renaming the options, from static=False => shared=True, and I am thinking whether it is really necessary to force zlib to have the same linkage as Protobuf. Maybe it could be left to the defaults, and let the downstream consumer to choose, maybe they want to link protobuf statically, but zlib dynamically? What do you think? Thanks!